baseplate/lib/__init__.py

```diff
@@ -45,3 +46,15 @@ def __get__(self, instance: T, owner: Type[T]) -> R:
         ret = self.wrapped(instance)
         setattr(instance, self.wrapped.__name__, ret)
         return ret
+
+
+class UnknownCallerError(Exception):
+    def __init__(self) -> None:
+        super().__init__("Could not determine calling module's name")
+
+
+def get_calling_module_name() -> str:
+    module = inspect.getmodule(inspect.stack()[2].frame)
```

Comment:
🔕 is there any way this can raise an `IndexError`?

Reply:
I don't want to say know cus I can't say for sure... but I hope not?